cloudformation/anghammarad.template.yaml

```diff
@@ -25,11 +25,35 @@ Mappings:
       Value: anghammarad
 
 Resources:
+
+  AnghammaradRole:
+    Type: AWS::IAM::Role
+    Properties:
+      RoleName: !Sub Anghammarad-${Stage}
+      AssumeRolePolicyDocument:
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                - lambda.amazonaws.com
+            Action:
+            - sts:AssumeRole
+      Path: /
+      Policies:
+        - PolicyName: LambdaPolicy
+          PolicyDocument:
+            Statement:
+              - Effect: Allow
+                Action:
+                - ses:SendEmail
+                Resource: "*"
+
   ProcessorFunction:
     Type: AWS::Serverless::Function
     Properties:
       FunctionName: !Sub anghammarad-${Stage}
       Description: !Sub Process Anghammarad ${Stage} notifications
+      Role: !GetAtt AnghammaradRole.Arn
```

Comment:
Might be easier to just provide the `Policy`, rather than the entire `Role`? If you give a `Policies` attribute all the policies you provide will be *added* to the default Lambda role so you only need to add the permissions that are being added, not the rest of the role (e.g. the assume role bit)

Reply:
https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md

ctrl+f for "Policies"

Reply:
Thanks - less YAML is better